babel.config.js

```diff
@@ -1,8 +1,8 @@
 module.exports = {
   presets: ['@babel/react'],
   plugins: [
-    '@babel/plugin-transform-runtime',
-    '@babel/plugin-proposal-class-properties',
+    ['@babel/plugin-transform-runtime', { loose: true }],
+    ['@babel/plugin-proposal-class-properties', { loose: true }],
```

Comment:
Did you check the dist / build after changing babel config? (I mean testing It with the projects we're using IUI in.)

Reply:
no,please do that if you have time

Reply:
@mohsenParvar I checked it and there's no need to add `{ loose: true }` to babel config.